herddb-core/src/main/java/herddb/sql/CalciteEnumUtils.java

```diff
@@ -0,0 +1,26 @@
+package herddb.sql;
+
+import org.apache.calcite.linq4j.JoinType;
+import org.apache.calcite.rel.core.JoinRelType;
+
+public class CalciteEnumUtils {
+
+    public static JoinType toLinq4jJoinType(JoinRelType joinRelType) {
```

Comment:
We should add an unit test with a query for every type of join.
You can add a breakpoint or a throw new RuntimeException() on every case clause and then run the tests about joins, this way you will see if there is some kind of join not covered by tests

Reply:
Sounds good. I will be adding these tests shortly.

Reply:
I will be adding UT for this tonight.